hbase-shaded/hbase-shaded-testing-util-tester/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <!--
+      /**
+       * Licensed to the Apache Software Foundation (ASF) under one
+       * or more contributor license agreements.  See the NOTICE file
+       * distributed with this work for additional information
+       * regarding copyright ownership.  The ASF licenses this file
+       * to you under the Apache License, Version 2.0 (the
+       * "License"); you may not use this file except in compliance
+       * with the License.  You may obtain a copy of the License at
+       *
+       *     http://www.apache.org/licenses/LICENSE-2.0
+       *
+       * Unless required by applicable law or agreed to in writing, software
+       * distributed under the License is distributed on an "AS IS" BASIS,
+       * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+       * See the License for the specific language governing permissions and
+       * limitations under the License.
+       */
+      -->
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.hbase</groupId>
+        <artifactId>hbase-build-configuration</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+        <relativePath>../../hbase-build-configuration</relativePath>
+    </parent>
+
+    <artifactId>hbase-shaded-testing-util-tester</artifactId>
+    <name>Apache HBase - Shaded - Testing Util Tester</name>
+    <description>Ensures that hbase-shaded-testing-util works with hbase-shaded-client.</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
```

Comment:
Should be test scope?

Reply:
I created a project which tries to be as similar as our user's project:
- logging and shaded-client are on compile classpath,
- shaded-testing-util is on test classpath.

Reply:
Ah, yeah, this is the "downstream" project for the testing-util, but it looks like this module doesn't have anything in `src/main/java` (much less stuff that uses slf4j's log4j), so `<scope>test</scope>` would make sense.

Reply:
Ok.